makefiles/vars.inc.mk

```diff
@@ -17,6 +17,7 @@ export USEMODULE             # Sys Module dependencies of the application. Set i
 export USEPKG                # Pkg dependencies (third party modules) of the application. Set in the application's Makefile.
 export DISABLE_MODULE        # Used in the application's Makefile to suppress DEFAULT_MODULEs.
 export APPDEPS               # Files / Makefile targets that need to be created before the application can be build. Set in the application's Makefile.
+# BUILDDEPS                  # Files / Makefile targets that need to be created before starting to build. Set by modules / build system included files.
```

Comment:
took me a while to parse "Set by modules / build system included files.". What does "Set by build system included files." mean? Maybe just drop the whole "Set by" phrase?

Reply:
I was writing in the same way as the line before but it does not help, I removed it.